framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
I guess the problem depends on who is the 'master' I would assume that the 'widget' is the master ... so if a widget is disabled then there should be no way of activating it ... if a command for activating the widget is disabled, then that should only affect the command and not the widget ... (and no I did not read the whole thread, only my reaction to reading derrels last comment)